back/src/test/java/com/cocktailpick/back/user/service/UserServiceTest.java

```diff
@@ -77,6 +78,15 @@ void setUp() {
 		user.setFavorites(favorites);
 	}
 
+	@Test
+	void updateUser() {
+		UserUpdateRequest userUpdateRequest = new UserUpdateRequest("작은곰");
+		userService.updateUser(user, userUpdateRequest);
+
+		assertThat(user.getName())
+			.isEqualTo(userUpdateRequest.getName());
```

Comment:
```suggestion
		assertThat(user.getName()).isEqualTo(userUpdateRequest.getName());
```
코드가 길지 않아서 줄바꿈이 없어도 될 것 같아요!

Reply:
```suggestion
		assertEquals(user.getName(), userUpdateRequest.getName());
```

요 메서드도 있지요😊

Reply:
토니 피드백으로 수정할게요~

Reply:
https://jongmin92.github.io/2020/03/31/Java/use-assertthat/

assertXXX 사용을 지양하자는 이야기가 있네요

Reply:
이 부분에 대해서 저도 오늘 제이슨에게 피드백을 들었고 하나로 통일하는 편이 좋다는 이야기를 들었네요.
이미 구현이 되었다면 놔두고 아직 리팩토링을 할 예정이라면 다시 원상복귀해도 좋을 것 같아요.
귀찮게하여 미안합니다😢